ps3mf/save.go

```diff
@@ -164,8 +168,8 @@ func (b *Bundle) Save(path string) (err error) {
 				GetMeta("Copyright", ""),
 				GetMeta("LicenseTerms", ""),
 				GetMeta("Rating", ""),
-				GetMeta("CreationDate", currentDate),
-				GetMeta("ModificationDate", currentDate),
+				GetMeta("CreationDate", CurrentDate()),
+				GetMeta("ModificationDate", CurrentDate()),
```

Comment:
Why not call the function once above (where the variable was originally) and use it here for DRYness?